src/require-strict-assert.spec.ts

```diff
@@ -0,0 +1,138 @@
+// require-strict-assert.spec.ts
+
+/*
+ * Copyright (c) 2021-2023 Check Digit, LLC
+ *
+ * This code is licensed under the MIT license (see LICENSE.txt for details).
+ */
+import { RuleTester } from 'eslint';
+
+import rule from './require-strict-assert';
+
+describe('require-strict-assert', () => {
+  const ruleTester = new RuleTester({ parserOptions: { ecmaVersion: 2020, sourceType: 'module' } });
+  ruleTester.run('require-strict-assert', rule, {
+    valid: [
+      {
+        code: `import { strict as assert } from 'node:assert';`,
+      },
+      {
+        code: `import { strict as assert } from 'node:assert';
+               import otherModule from 'other-module';`,
+      },
```

Comment:
we're not testing for valid method usage (i.e. `assert.equal()` etc), should be added.

Reply:
also, tests for things that aren't assert - e.g. `foo.strictEqual()` should be fine.  Not sure if that's being dealt with correctly in the logic of the rule.

Reply:
> we're not testing for valid method usage (i.e. `assert.equal()` etc), should be added.

We have the coverage for valid method usage at the bottom

Reply:
the test i'm talking about is:
```
import foo from 'something-that-is-not-assert';
const val1 = 'val2';
foo.strictEqual(val1, 'val2')
```
i think this might get "fixed" incorrectly.